guides/overview.md

```diff
@@ -4,9 +4,135 @@ Amber is designed to make common web-development tasks simple, and easy, maintai
+
+Amber is opinionated software. It makes the assumption that there is a "best" way to do things, and it's designed to encourage that way - and in some cases to discourage alternatives.
+
+The Amber philosophy includes two major guiding principles:
+
+* **Don't Repeat Yourself:**
+  DRY is a principle of software development which states that "Every piece of knowledge must have a single, unambiguous, authoritative representation within a system." By not writing the same information over and over again, our code is more maintainable, more extensible, and less buggy.
+* **Convention Over Configuration:**
+  Amber provides a basic organizational structure that covers class names, filenames, database table names, and other conventions. While the conventions take some time to learn, by following the conventions Amber provides you can avoid needless configuration and make a uniform application structure that makes working with various projects simple.
+
+## Creating a New Amber Project
+
+Open up a command line prompt. On macOS open Terminal.app, on Windows choose "Run" from your Start menu and type `cmd.exe`. Any commands prefaced with a dollar sign `$` should be run in the command line. Verify that you have a current version of Ruby installed:
+
+```bash
+$ crystal -v
+Crystal 0.23.1 (2017-09-08) LLVM 4.0.1
+```
+
+Many popular UNIX-like OSes ship with an acceptable version of SQLite3. On Windows, if you installed Rails through Rails Installer, you already have SQLite installed. Others can find installation instructions at the [SQLite3 website](https://www.sqlite.org/). Verify that it is correctly installed and in your PATH
+
+```bash
+$ sqlite3 --version
+```
+
+The program should report its version.
+
+### Installing Amber
+
+To install the Amber binary on Mac you can use
+
+```bash
+$ brew install amberframework/amber/amber
+```
+
+To verify that you have everything installed correctly, you should be able to run the following:
+
+```bash
+$ amber -v
+Amber CLI (amberframework.org) - vX.X.X
+```
+
+For other Linux distributions and guides please see Installation
+
+### Creating the Blog Application
+
+Amber comes with a number of scripts called generators that are designed to make your development life easier by creating everything that's necessary to start working on a particular task. One of these is the new application generator, which will provide you with the foundation of a fresh Amber application so that you don't have to write it yourself.
+
+To use this generator, open a terminal, navigate to a directory where you have rights to create files, and type:
+
+```bash
+$ amber new blog -d sqlite
+```
+
+This will create an Amber application called Blog in a `blog` directory using the sqlite database engine. If you don't specify a database the default will be `postgresql`
+
+Tip: You can see all of the command line options that the Rails application builder accepts by running `rails new -h`
```

Comment:
Tip: You can see all of the command line options that `amber new` accepts by running `amber new -h`

Based on @marksiemers comment here: https://github.com/amberframework/docs/pull/43/files#r154251682